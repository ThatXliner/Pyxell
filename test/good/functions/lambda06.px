
func call(Bool->String->Void f) def
    f(true, "$")

func f(Int a, Bool b, Char c, String d: "---") def
    print a, b, c, d

call(lambda b, d -> f(6, b, 'a'))
call(f(42, _, d=_, c='x'))
