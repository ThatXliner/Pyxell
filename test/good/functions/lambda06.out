6 true a ---
42 true x $
